cluster/common.sh

```diff
@@ -59,10 +59,13 @@ function create-kubeconfig() {
 
   export KUBECONFIG=${KUBECONFIG:-$DEFAULT_KUBECONFIG}
   # KUBECONFIG determines the file we write to, but it may not exist yet
-  if [[ ! -e "${KUBECONFIG}" ]]; then
-    mkdir -p $(dirname "${KUBECONFIG}")
-    touch "${KUBECONFIG}"
-  fi
+  for cfg in ${KUBECONFIG//:/ } ; do
```

Comment:
I think it would be clearer to set `IFS=':'` in the context of the loop - it's functionally equivalent but in my personal opinion clearer - you're declaring that you want to iterate in a specific way instead of indirectly with the substitution. That's an opinion, though.